templates/error/descriptions/E0003.md.hbs

```diff
@@ -0,0 +1,15 @@
+# Trying to authenticate an unknown client
+
+The server received a request from a client with ID **{{ client_id }}**,
+but this client is not registered with the server.
+
+This is quite often caused by the server data being wiped,
+thereby forgetting all clients.
+After clients register they often cache this data in the browser
+and automatically try to reconnect to the server,
+but since the server no longer knows the client this request is rejected.
```

Comment:
```suggestion
When clients register, they often cache this data in the browser,
and try to automatically reconnect to the server,
but since the server no longer knows the client, this request is rejected.
```